cli/build_collection.go

```diff
@@ -224,8 +225,15 @@ func (o *BuildCollectionOptions) Run(ctx context.Context) error {
 
 	descs = append(descs, linkedDescs...)
 
-	// Store the DataSetConfiguration file in the manifest config of the OCI artifact for
-	// later use.
+	// Sort the slices within the DataSetConfiguration and store the file
```

Comment:
Can we move this logic to `config/load`? It would probably be useful to have these sorted upfront.

Reply:
I'll get on it